File: java/src/main/java/com/iteso/nintendo/attacks/ThunderWave.java
package com.iteso.nintendo.attacks;

public class ThunderWave implements Attack {
    static int damage;
    static String name = "Thunder Wave";

    @Override
    public String attackName() {
        return name;
    }

    @Override
    public int attackDamage() {
        return this.damage;
    }

    @Override
    public void setDamage(int damage) {
        this.damage = damage;
    }
}

File: python/ThunderWave.py
class ThunderWave:
    def __init__(self):
        self.damage = None
        self.name = 'Thunder Wave'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
